### PR TITLE
Updated gettext functions

### PR DESCRIPTION
**Pull request by dnaluz**
Updated the gettext functions for members. Please review, make sure all gettext functions use single quotes and instead of print __ we use _e to echo values out if assigning to a variable __ is ok. Make sure all functionality still works and displays all messages for creating, editing and viewing a person.

### CODE REVIEW

File: buddypress/members/index-directory.php [hunk 1]
```diff
@@ -775,30 +775,30 @@
...
         <div class="members__container">
             <div class="members__filter-container members__filter-container--hidden">
-                <span><?php print __("Search criteria:", "community-portal"); ?></span>
+                <span><?php _e('Search criteria:', 'community-portal'); ?></span>
                 <div class="members__select-container">
-                    <label class="members__label"><?php print __("Location", "community-portal"); ?></label>
+                    <label class="members__label"><?php _e('Location', 'community-portal'); ?></label>
                     <select class="members__location-select">
-                        <option value=""><?php print __('Select', "community-portal"); ?></option>
+                        <option value=""><?php _e('Select', "community-portal"); ?></option>
```
Comment (kycapstick) [inline buddypress/members/index-directory.php:791]: Community Portal needs to be changed to single-quotes

File: buddypress/members/single/profile.php [hunk 2]
```diff
@@ -247,7 +247,7 @@
                             <path d="M8 9.66602C9.10457 9.66602 10 8.77059 10 7.66602C10 6.56145 9.10457 5.66602 8 5.66602C6.89543 5.66602 6 6.56145 6 7.66602C6 8.77059 6.89543 9.66602 8 9.66602Z" stroke="#737373" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
                         </svg>
                         <?php if($location->location_country === 'OE'): ?>
-                          <?php print __("Online Event", "community-portal"); ?>
+                          <?php _e("Online Event", 'community-portal'); ?>
```
Comment (kycapstick) [inline buddypress/members/single/profile.php:250]: Quotes around Online Event need to be changed to single-quote

File: buddypress/members/index-directory.php (hunk 1 above)
Comment (dnaluz) [reply buddypress/members/index-directory.php:791]: Updated

File: buddypress/members/single/profile.php (hunk 2 above)
Comment (dnaluz) [reply buddypress/members/single/profile.php]: Updated

**Pull request merged**